pkg/api/v1/routes/handlers.go

```diff
@@ -136,6 +135,10 @@ func (r *Routes) serverConditionUpdate(c *gin.Context) (int, *v1types.ServerResp
 	return http.StatusOK, &v1types.ServerResponse{Message: "condition updated"}
 }
 
+// XXX: This needs to be refactored. In order to specifically compute the NATS subject for
```

Comment:
The ServerID and conditionKind is known and should suffice to compute the NATS subject without necessarily caring about the json.RawMessage payload, no?

Reply:
This isn't possible in the current state, so I'm leaving it as-is and we'll address in follow-on work.